src/App.js

```diff
@@ -1,24 +1,61 @@
-import logo from './logo.svg';
+import * as React from 'react';
 import './App.css';
+import { Container } from 'react-bootstrap';
+import { FormControl, MenuItem, Select } from '@mui/material';
 
 function App() {
+  const [club, setClub] = React.useState('');
+
+  const handleChange = (event) => {
+    setClub(event.target.value);
+    console.log(club);
+  };
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App" style={{ backgroundColor: "rgb(48, 185, 253)"}}>
+      <h1 style={{ fontFamily: 'courier'}}>Clubs Menu</h1>
+      <div className="content" style={{ fontFamily: 'courier'}}>
+        
+      </div>
+      <Container>
+        <FormControl>
+        <h1 style={{ fontFamily: 'courier'}}>Club Choice:</h1>
+          <Select
+            labelId="demo-simple-select-label"
+            id="demo-simple-select"
```

Comment:
Same thing as above, name this something more descriptive. These ID values mean nothing to most users because it does not pop up in the user interface but there are people with disabilities that rely on screenreaders. In the screenreaders, ID values are heavily used.